src/components/Table/utils/RowState.ts

```diff
@@ -10,30 +11,30 @@ import { GridDataRow, KEPT_GROUP, reservedRowKinds, SelectedState } from "src";
 export class RowState {
   /** Our row, only ref observed, so we don't crawl into GraphQL fragments. */
   row: GridDataRow<any>;
-  /** Our parent RowState, or the `header` RowState if we're a top-level row. */
-  parent: RowState | undefined;
```

Comment:
I was able to remove `parent` as unnecessary anymore; just simplifies things to have parents only point to their children, and not try to maintain the two-way connection.